Backend/app/rider-platform/rider-app/Main/spec/Storage/Booking.yaml

```diff
@@ -374,3 +387,58 @@ Booking:
     riderId: "!SecondaryKey" # forcing to be a secondary key
     quoteId: "!SecondaryKey" # forcing to be a secondary key
     riderTransactionId: "!SecondaryKey" # forcing to be a secondary key
+
+BookingPartiesLink:
+  fields:
+    id: Id BookingPartiesLink
+    bookingId: Id Booking
+    partyId: Id Person
+    partyType: TripParty|NoRelation
+    partyName: Text
+    isActive: Bool
+
+  excludedFields: [merchantId, merchantOperatingCityId]
+
+  constraints:
+    id: PrimaryKey
+    partyId: SecondaryKey
+    isActive: SecondaryKey
```

Comment:
Can be removed and query can be made as conditionalKV

Reply:
✅